xmrstak/pools.tpl

```diff
@@ -22,7 +23,7 @@ POOLCONF],
  *
  *    aeon7 (use this for Aeon's new PoW)
  *    bbscoin (automatic switch with block version 3 to cryptonight_v7)
- *    bittube (uses cyrptonight_bittube2 algorithm)
+ *    bittube (uses cryptonight_bittube2 algorithm)
```

Comment:
could you please provide this as separate PR. We try to avoid different topics in one PR. This is than much better for referencing in case of issues and also writing change logs.

Reply:
I applied it in the other patch under the Spelling Police so it can just be dropped here, for now

Depends which order you will be applying - I kept the other patchset atomic for whitespace / spelling / actual code separately